ethereum/core/src/main/java/org/hyperledger/besu/ethereum/trie/diffbased/common/trielog/TrieLogPruner.java

```diff
@@ -55,14 +61,26 @@ public TrieLogPruner(
       final Consumer<Runnable> executeAsync,
       final long numBlocksToRetain,
       final int pruningLimit,
-      final boolean requireFinalizedBlock) {
+      final boolean requireFinalizedBlock,
+      final MetricsSystem metricsSystem) {
     this.rootWorldStateStorage = rootWorldStateStorage;
     this.blockchain = blockchain;
     this.executeAsync = executeAsync;
     this.numBlocksToRetain = numBlocksToRetain;
     this.pruningLimit = pruningLimit;
     this.loadingLimit = pruningLimit; // same as pruningLimit for now
     this.requireFinalizedBlock = requireFinalizedBlock;
+    this.addedToPruneQueueCounter =
+        metricsSystem.createCounter(
+            BesuMetricCategory.PRUNER,
+            "trie_log_added_to_prune_queue",
+            "trie log added to prune queue");
+    this.prunedFromQueueCounter =
+        metricsSystem.createCounter(
+            BesuMetricCategory.PRUNER, "trie_log_pruned_from_queue", "trie log pruned from queue");
+    this.prunedOrphanCounter =
+        metricsSystem.createCounter(
+            BesuMetricCategory.PRUNER, "trie_log_pruned_orphan", "trie log pruned orphan");
```

Comment:
these metric names are only meaningful if you already understand the trie log pruning implementation.  I would suggest trie_log_pruned_head and trie_log_pruned_forks

even that implies an understanding of canonical chain and forks.  but I had to look at the code to figure out what we meant by _orphan and _queue.

yay, naming feedback.